tests/integration/routes/test_view_submission.py

```diff
@@ -29,10 +29,6 @@ def setUp(self):
 
     def test_view_submission(self):
```

Comment:
I've just noticed that this test is no longer asserting anything, which suggests it should just be completely removed.